pkg/serializer/serializer_test.go

```diff
@@ -159,15 +190,43 @@ func registerNewCRD(scheme *runtime.Scheme) error {
 var _ crdconversion.Hub = &CRDNewVersion{}
 
 type CRDNewVersion struct {
-	runtimetest.ExtensionB
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata"`
+	OtherString       string `json:"otherString"`
 }
 
 func (t *CRDNewVersion) Hub() {}
```

Comment:
Could add a comment like `Mark CRDNewVersion has the Hub type to become a conversion target`